photon-core/src/main/java/org/photonvision/vision/frame/consumer/MJPGFrameConsumer.java

```diff
@@ -176,6 +213,13 @@ public void accept(Frame frame) {
         }
     }
 
+    // ------------------------------------------------------------------------------------
+    //
+    //  This is called if video streaming is disallowed.  It sets a disabled flag.
+    //  A "Streaming Disabled" picture will be sent to the stream source once a second.
+    //  With implementation of the AnyConnectedClients, this should only be called
+    //  when there are no active stream clients (dashboards).
+    //
```

Comment:
If you're going to add comments for methods/classes please follow Javadoc guidelines for them. Many IDEs will pre-generate most of the formatting for you.